doc/association-rules.rst

```diff
@@ -0,0 +1,83 @@
+Association Rules
+=================
+
+.. figure:: icons/association-rules.png
+
+Induction of association rules.
+
+Signals
+-------
+
+#### Inputs
+
+-  **Data**
+
+   Data set
+
+#### Outputs
```

Comment:
NP. That's why I was thoroughly confused, because we did all the former docs in official .rst markdown (and in proper .md before that). Will follow .rst from now on.